### PR TITLE
Suggestions and minor edits.

### PR DESCRIPTION
**Pull request by elainegrace**
Deleted and closed the previous pull request, added the suggestions and edits as inline comments on these commits.

### CODE REVIEW

File: manuscript/react-modern/meet-the-react-component.md [hunk 1]
```diff
@@ -49,18 +51,19 @@ function App() {
...
-* Second, the App component doesn't receive any parameters in its function signature yet. In the upcoming sections, you will learn how to pass information (props) from one component to another component. These props will be accessible via the function's signature as parameters then. (see **props** later)
-* And third, the App component returns code that resembles HTML. You will see how this new syntax, called JSX, allows you to combine JavaScript and HTML for displaying highly dynamic and interactive content in a browser. (see **JSX** later)
+- First, this React component, called App component, is just a JavaScript function. In contrast to JavaScript functions, it's defined in [PascalCase](https://www.robinwieruch.de/javascript-naming-conventions). This kind of component is commonly called **function component**. Function components are the modern way of using components in React, however, be aware that there are other variations of React components too. (see **component types** later)
```
Comment (elainegrace) [inline manuscript/react-modern/meet-the-react-component.md]: Lines 56, 57, 58: Suggestion to add link to the **component types**, **props**, **JSX** for easy content navigation.

File: manuscript/react-modern/lists.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ The map function is inlined concisely in your JSX. Within the map function, we h
   * Confirm the [changes](https://github.com/the-road-to-learn-react/hacker-stories/compare/2021/React-JSX...2021/Lists-in-React).
 * Read more about why React's key attribute is needed ([0](https://dev.to/jtonzing/the-significance-of-react-keys---a-visual-explanation--56l7), [1](https://www.robinwieruch.de/react-list-key), [2](https://reactjs.org/docs/lists-and-keys.html)). Don't worry if you don't understand the implementation yet, just focus on what problem it causes for dynamic lists.
 * Recap the [standard built-in array methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/) -- especially *map*, *filter*, and *reduce* -- which are available in native JavaScript.
-* What happens if your return `null` instead of the JSX? Try it first, then read the answer:
+* What happens if you return `null` instead of the JSX? Try it first, then read the answer:
```
Comment (elainegrace) [inline manuscript/react-modern/lists.md:186]: Typo

**Review by elainegrace — COMMENTED** [review]
image

A suggestion on this part of chapter Lists in React (page 19), while the highlighted code snippet works, it is probably a good idea to include in the explanation below to have the readers' Browser tools-Console open. It can be seen early on, the console advises on having a unique key for every child in a list. The importance of building the early habit of setting unique keys would probably be instilled better to readers as to why the current implementation could be improved.

File: manuscript/react-modern/component-definition.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ const List = () => {
 };
 ~~~~~~~
 
-If an arrow function doesn't do *anything* in between, but only returns *something*, -- in other words, if an arrow function doesn't have business logic in between, but only returns --, you can remove the **block body** (curly braces) of the function. In a **concise body**, an **implicit return statement** is attached, so you can remove the return statement:
+If an arrow function’s only purpose is to return a value and it doesn’t have any business logic in between, you can remove the **block body** (curly braces) of the function. In a **concise body**, an **implicit return statement** is attached, so you can remove the return statement:
```
Comment (elainegrace) [inline manuscript/react-modern/component-definition.md:101]: Rewording suggestion for a concise description.

**Review by elainegrace — COMMENTED** [review]
image

A suggestion on this last part of the chapter **Spread and Rest Operators**:
For a better takeaway, better content navigation, and quick access to the points of comparison, consider adding reference links for the **variation 1** and **variation 2** code snippets.

**Comment by rwieruch** [comment]
Merged in here https://github.com/the-road-to-learn-react/the-road-to-react/commit/ac752c7aea3bafa199ba4309a1c2914f659f9d47 with more recommendations by @javelwilson 🚀 🎉

**Pull request closed without merging**